pages/test_analytics/javascript_collectors.md.erb

```diff
@@ -1,6 +1,6 @@
 # JavaScript Collectors
```

Comment:
This is my first encounter on a rename with the Buildkite docs. What do we need to do to make sure links to the former URL redirect to the new one?

Reply:
Either update all the links manually or set redirects in routes.rb @ddbeck

Reply:
Oh great, `routes.rb` is where it's at! [Cool URIs don't change](https://www.w3.org/Provider/Style/URI).